xpsi/Instrument.py

```diff
@@ -213,6 +213,14 @@ def energy_edges(self):
     def energy_edges(self, energy_edges):
         """ Set the energy edges in keV. """
 
+        try:
```

Comment:
try-except blocks like this are ugly, but remnant of what I chose a while ago to reduce LOC. Is there a more elegant way than asserts in try-except block?